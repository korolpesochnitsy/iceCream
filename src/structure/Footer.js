import React from 'react';
import { css } from 'emotion/macro';

const footerStyle = css`
  max-width: 63.75em;
  margin-left: auto;
  margin-right: auto;
  border-radius: 0 0 4px 4px;
  color: #313030;
  text-align: center;
  padding-bottom: 2em;

  span {
    margin: 0;
    font-family: 'cornerstone', sans-serif;
    font-weight: 600;
  }
`;

const Footer = () => (
  <footer className={footerStyle}>
    <span>&copy; Ice Cream</span>
  </footer>
);

export default Footer;
