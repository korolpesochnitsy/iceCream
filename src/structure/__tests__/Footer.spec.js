import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer', () => {
  it('should render', () => {
    const { container } = render(<Footer />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
