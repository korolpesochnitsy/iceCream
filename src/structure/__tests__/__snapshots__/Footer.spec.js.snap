// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
.emotion-0 {
  max-width: 63.75em;
  margin-left: auto;
  margin-right: auto;
  border-radius: 0 0 4px 4px;
  color: #313030;
  text-align: center;
  padding-bottom: 2em;
}

.emotion-0 span {
  margin: 0;
  font-family: 'cornerstone',sans-serif;
  font-weight: 600;
}

<footer
  class="emotion-0"
>
  <span>
    Â© Ultimate Ice Cream
  </span>
</footer>
`;
